### PR TITLE
check service enable state in test mode

### PR DESCRIPTION
**Pull request by clan**
add missing service enabled/disabled state check

### CODE REVIEW

File: salt/states/service.py [hunk 1]
```diff
@@ -385,13 +384,12 @@ def dead(name, enable=None, sig=None, **kwargs):
...
-            elif enable is False and before_toggle_enable_status:
-                ret.update(_disable(name, None, **kwargs))
-            return ret
-        else:
+        if enable is True and not before_toggle_enable_status:
+            ret.update(_enable(name, None, **kwargs))
+        elif enable is False and before_toggle_enable_status:
+            ret.update(_disable(name, None, **kwargs))
+        return ret
+        if __opts__['test']:
```
Comment (cachedout) [inline salt/states/service.py]: This code is unreachable. Indention typo?

**Comment by cachedout** [comment]
Hi @clan. There's some unreachable code that I've commented on. Also, it does not look like tests are passing. Could you please take a look?

**Comment by clan** [comment]
fixed, and repushed

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by clan** [comment]
no error log for centos 6 & 7, it seems like other problem unrelated to this pull request?

**Comment by cachedout** [comment]
Yup, unrelated. I'm going to go ahead and merge this now.

**Pull request merged**